gradle/versions.gradle

```diff
@@ -17,7 +17,7 @@ ext {
   mockitoVersion = '2.10.0'
 
   // ~ benchmark dependencies
-  beamVersion = '0.6.0'
+  beamVersion = '2.7.0'
```

Comment:
Should the version be 2.8.0?

Reply:
If 2.8.0 would be released then yes :), but its not yet